dbt/models/proximity/proximity.dist_pin_to_traffic_master.sql

```diff
@@ -0,0 +1,340 @@
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH traffic_minor AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Minor Arterial'
+),
+
+traffic_interstate AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Interstate'
+),
+
+traffic_freeway AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Freeway and Expressway'
+),
+
+traffic_local AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Local Road or Street'
+        AND daily_traffic IS NOT NULL
+),
+
+traffic_collector AS (  -- noqa: ST03
+    SELECT *
+    FROM {{ source('spatial', 'traffic') }}
+    WHERE road_type = 'Major Collector'
+        OR road_type = 'Minor Collector'
```

Comment:
Minor collectors aren't present in all years and create error messages on joins.